polkadot/xcm/pallet-xcm/src/precompiles/IXcm.sol

```diff
@@ -1,26 +1,28 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-/// @dev The XCM Precompile contract address.
+/// @dev The on-chain address of the XCM (Cross-Consensus Messaging) precompile.
 address constant XCM_PRECOMPILE_ADDRESS = address(0xA0000);
 
-/// @title An interface for interacting with `pallet_xcm`
-/// @notice Provides functions for executing and sending XCM messages.
-/// Learn more about XCM: https://docs.polkadot.com/develop/interoperability/
-/// @dev All parameters must be encoded using the SCALE codec.
+/// @title XCM Precompile Interface
+/// @notice A low-level interface for interacting with `pallet_xcm`.
+/// It forwards calls directly to the corresponding dispatchable functions, providing access to XCM execution and message passing.
```

Comment:
We should avoid making the doc string sentences too long  ideally each line should be less than 120 characters

```solidity
/// It forwards calls directly to the corresponding dispatchable functions, providing access to XCM execution and message passing.
```

should be

```solidity
/// It forwards calls directly to the corresponding dispatchable functions.
/// Providing access to XCM execution and mess
```

WDYT @tiagobndr